packages/xrpl/test/models/AMMBid.test.ts

```diff
@@ -0,0 +1,168 @@
+import { assert } from 'chai'
+
+import { validate, ValidationError } from '../../src'
+import { validateAMMBid } from '../../src/models/transactions/AMMBid'
+
+/**
+ * AMMBid Transaction Verification Testing.
+ *
+ * Providing runtime verification testing for each specific transaction type.
+ */
+describe('AMMBid', function () {
+  let bid
+
+  beforeEach(function () {
+    bid = {
```

Comment:
Should this go into the fixtures file?

Reply:
Actually, seeing the test - seeing the original fields is very helpful, so I don't think it should.